feature(about): Complete about page

ernestognw opened the pull request:
## Summary
About page was completed with general information about the project and contributors. Also, some changes in layout and presentation files with the name of the dashboard.

#### Note
There is no need to update tests since it is almost static info, which render is already tested in a past test. 

## Screenshot
![localhost_3000_about](https://user-images.githubusercontent.com/33379285/91345002-55e5d580-e7a4-11ea-85c4-1fdaaa5dfc0a.png)

SofiaVega commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+            </Typography>
+            <Typography gutterBottom variant="body1">
+              Because of that, it is a problem when Chrome is going to launch a
+              new feature or to deprecate any functionality. That&lsquo;s why we
```
I think 'is going to launch a new feature or deprecate any functionality' (remove the word 'to') sounds better

ernestognw replied:
Done!

SofiaVega commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+            </Typography>
+            <Typography gutterBottom variant="body1">
+              The Chrome Security Dashboard is intended to present public data
+              about future changes to be done in future Chrome Versions, and to
```
Future changes to be done in future Chrome versions sounds a bit redundant, you could remove the first 'future'

ernestognw replied:
Done!

SofiaVega commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+              evaluated many options of public data sources and ended up using
+              HTTP Archive public data sets hosted on BigQuery service on Google
+              Cloud. This data is public and accessible from all over the world,
+              which give us enough confidence to use it as a reference to make
```
I think its 'gives us enough confidence' (add an 's')

ernestognw replied:
Done!

SofiaVega commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+            <Typography gutterBottom variant="body1">
+              As part of our effort to show enough data to support decisions, we
+              evaluated many options of public data sources and ended up using
+              HTTP Archive public data sets hosted on BigQuery service on Google
```
Is it data sets or datasets? You use one here and a different one below

ernestognw replied:
I decided to go for `datasets`, but I'll keep my attention to what Livvie or Carlos comment

JonathanChavezTamales commented:
+1 to what @SofiaVega mentioned, it LGTM! Good job Ernesto.

ernestognw commented:
> Looks great!! I think it would be nice if Livvie or Carlos could approve the first text just to be sure. Other than that I added some comments about the text (repeated words, etc)

Thanks for your comments Sofi, I addressed them, but also I will wait for approval from Livvie or Carlos

ernestognw commented:
> Looks great!! I think it would be nice if Livvie or Carlos could approve the first text just to be sure. Other than that I added some comments about the text (repeated words, etc)

Thanks for your comments Sofi, I addressed them, but also I will wait for approval from Livvie or Carlos

carlosjoan91 commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+            <Typography gutterBottom variant="body1">
+              As part of our effort to show enough data to support decisions, we
+              evaluated many options of public data sources and ended up using
+              HTTP Archive public datasets hosted on BigQuery service on Google
```
I would remove the "hosted on BigQuery service on Google Cloud" part, since for a general user, the meaningful part is what dataset it's from, not so much how it's retrieved.

ernestognw replied:
Done!

ernestognw commented on src/views/about/index.js:
```diff
@@ -12,9 +29,132 @@ import {Typography} from '@material-ui/core';
+            <Typography gutterBottom variant="body1">
+              The Chrome Security Dashboard is intended to present public data
+              about changes to be done in future Chrome Versions, and to serve
+              as a tool to negotiate next steps with other browsers.
```
Done!
